### ISSUE TITLE
Armorstands spawned with "/npc create <name> --type armorstand" can't be removed properly

### ISSUE THREAD

**Issue by axemwa** [CitizensDev/Citizens2#1681]
There's some armorstands I spawned with "/npc create <name> --type armorstand" that I can't remove properly and by that I mean that I can't remove the Displayname that is where the armorstand used to be.
The armorstands were duplicates.
The armorstands themselves were definately removed though as they can no longer be seen with spectator mode on and are no longer in the saves.yml... HOWEVER the Displayname Still remains

The output of command `/version` on my server is: **This server is running Paper version git-Paper-509 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)**

The output of command `/version citizens` on my server is: **[14:39:09 INFO]: Citizens version 2.0.24-SNAPSHOT (build 1612)
[14:39:09 INFO]: Website: http://www.citizensnpcs.co
[14:39:09 INFO]: Authors: aPunch and fullwall**

Ps: This photo was taken in spectator mode to show u the armor stand isnt there but the displayname still is... The armorstand in the picture used to be one of those smalll armorstands before I removed them, they also had their gravity set to off and were floating
![image](https://user-images.githubusercontent.com/37608633/51796461-507c7e00-2243-11e9-87cf-978b9ff46db9.png)
The displayname for the removed armorstand is "&c&lYou are here" as u can see in pic

**Comment by mcmonkey4eva** [comment]
If the NPC is gone, the actual entity is gone, and there's just a floating nameplate... that sounds like an issue in Minecraft or some clientside mod.

**Comment by axemwa** [comment]
I'm definately not using any mods, I used to use forge mods but forge isn't out for 1.13 yet as far as I know.
Are u sure it didn't attempt make a second armor stand or something and bug out? Because remember that glitch with npcs where u can select the NPC's armor stand (for sitting etc) when u type /npc sel instead of selecting the actual NPC it will select a thing called "!"? Well I was thinking it could have something to do with that maybe but I'm probs wrong its just an idea. Also these were mini-armorstands so I don't know if they might have a bug with them or not. The effects I applied to the armorstands was gravity off so they float and with invisibility on. I did this so that it indicates to players where they are on the map. The floating text is still useful to me for now but is there anyway I can remove it with a command or something for future reference, maybe worldedits //regen command to regenerate the affected area or would there be something more simple than that?
